guides/common/assembly_preparing-environment-for-satellite-installation-ipv6.adoc

```diff
@@ -0,0 +1,9 @@
+[id="preparing-environment-for-{project}-installation-ipv6"]
```

Comment:
Do you really need to mention the project in the ID? Isn't it better to avoid it when possible?

Reply:
That's a good point, it is probably better to avoid using project in the ID. However, in some cases there will be similar modules for Foreman and Smart Proxy. In this cases we need to specify this in the ID anyway. For example, in `proc_installing-the-satellite-server-packages.adoc` and `proc_installing-capsule-server-packages.adoc`.

Reply:
Agreed, but I'd suggest to leave it out when you can. It can even avoid confusion. Here the user may be looking for a document to prepare the capsule but as I read this, it's more of an integral env preparation.

Reply:
Another discussion we need to have. The obvious solution is to remove these from all IDs however this will likely break links in existing Satellite docs. Maybe we can add two IDs to those places with a comment that the old one is legacy for keeping links alive.